ledger-core/scripts/insolard/launchnet.sh

```diff
@@ -12,7 +12,7 @@ INSOLAR_LOG_FORMATTER=${INSOLAR_LOG_FORMATTER:-"json"}
 INSOLAR_LOG_LEVEL=${INSOLAR_LOG_LEVEL:-"debug"}
 # we can skip build binaries (by default in CI environment they skips)
 SKIP_BUILD=${SKIP_BUILD:-${CI_ENV}}
-BUILD_TAGS=${BUILD_TAGS:-''}
+BUILD_TAGS=${BUILD_TAGS:-'debug'}
```

Comment:
is it OK for everyone? i remember we have some discussion about default options for launchnet, @ivanshibitov i suppose you was involved in it?

Reply:
before it has convlogtxt, which I didn't like. debug is nice to have and has close to 0 performance effect